### PR TITLE
Change CLA bot link from ``the-knights-who-say-ni`` to ``ambv/cla-bot``

### PR DESCRIPTION
**Pull request by DanielNoord**
Since this tool is archived and no longer used I don't think it should be in the README anymore.

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Perhaps add the new CLA bot to the list?

**Comment by DanielNoord** [comment]
> Perhaps add the new CLA bot to the list?

I wasn't sure which to add.

https://github.com/apps/cpython-cla-bot doesn't tell you which repository it's working from. It seems like it is https://github.com/ambv/cla-bot but I also saw posts about that repo being move to the `python` org... Not sure where to link to at the moment.

**Comment by hugovk** [comment]
Yeah, it's at https://github.com/ambv/cla-bot.

I don't know when it's being moved, but when it is, the old link should redirect to the new location.

File: README.rst
Comment (DanielNoord) [inline README.rst:43]:
```suggestion
                                        projects.
```

File: README.rst
Comment (DanielNoord) [inline README.rst:54]:
```suggestion
.. _`ambv/cla-bot`: https://github.com/ambv/cla-bot
```

File: README.rst
Comment (hugovk) [inline README.rst]:
```suggestion
.. _`ambv/cla-bot`: https://github.com/ambv/cla-bot
```

**Review by hugovk — APPROVED** [review]
Thank you!

File: README.rst
Comment (ezio-melotti) [inline README.rst:42]: In the "Issue tracker" column you could add the upstream link (https://github.com/edgedb/cla-bot/issues) or a link to this repo (https://github.com/python/core-workflow/issues).  @ambv, do you have any opinion on this?

Comment (ambv) [reply README.rst:42]: We will move ambv/cla-bot under python/cla-bot when I'm back from my July travels. I'm at EuroPython right now, are you?

Comment (ezio-melotti) [reply README.rst:42]: Unfortunately not.  Do you want to wait until the bot is moved to update and merge this?

Comment (ambv) [reply README.rst:42]: We can merge as-is and then update once it's moved.

Comment (ezio-melotti) [reply README.rst:42]: I merged this and created #470 as a reminder for the update.

**Pull request merged**
